In [20]:
# All Libraries required

!pip install numpy==2.0.2
!pip install pandas==2.2.2
!pip install tensorflow_cpu==2.18.0
!pip install scikit-learn==1.3.1

  Using cached numpy-2.0.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
Using cached numpy-2.0.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (19.5 MB)
  Attempting uninstall: numpy
    Found existing installation: numpy 1.26.4
    Uninstalling numpy-1.26.4:
      Successfully uninstalled numpy-1.26.4
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
cupy-cuda12x 12.2.0 requires numpy<1.27,>=1.20, but you have numpy 2.0.2 which is incompatible.
gensim 4.3.3 requires numpy<2.0,>=1.18.5, but you have numpy 2.0.2 which is incompatible.
langchain 0.3.12 requires numpy<2,>=1.22.4; python_version < "3.12", but you have numpy 2.0.2 which is incompatible.
matplotlib 3.8.0 requires numpy<2,>=1.21, but you have numpy 2.0.2 which is incompatible.
pytensor 2.26.4 requires numpy<2,>=1.17.0, but you have numpy 2.0.2 which is incompa

In [21]:
import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split
from sklearn.metrics import mean_squared_error
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense

In [36]:
# Load dataset
data = pd.read_csv("/content/drive/MyDrive/ML/concrete_data.csv")

In [37]:
# Split features and target
X = data.drop(columns=["Strength"])
y = data["Strength"]

In [38]:
# Normalize the predictors manually using the formula
X_normalized = (X - X.mean()) / X.std()

In [39]:
print(X_normalized)

        Cement  Blast Furnace Slag   Fly Ash     Water  Superplasticizer  \
0     2.476712           -0.856472 -0.846733 -0.916319         -0.620147   
1     2.476712           -0.856472 -0.846733 -0.916319         -0.620147   
2     0.491187            0.795140 -0.846733  2.174405         -1.038638   
3     0.491187            0.795140 -0.846733  2.174405         -1.038638   
4    -0.790075            0.678079 -0.846733  0.488555         -1.038638   
...        ...                 ...       ...       ...               ...   
1025 -0.045623            0.487998  0.564271 -0.092126          0.451190   
1026  0.392628           -0.856472  0.959602  0.675872          0.702285   
1027 -1.269472            0.759210  0.850222  0.521336         -0.017520   
1028 -1.168042            1.307430 -0.846733 -0.279443          0.852942   
1029 -0.193939            0.308349  0.376762  0.891286          0.400971   

      Coarse Aggregate  Fine Aggregate       Age  
0             0.862735       -1.2170

In [40]:
# Initialize list to store MSE values
mse_list = []

# Repeat 50 times
for _ in range(50):
    # 1. Split the data into training and test sets
    X_train, X_test, y_train, y_test = train_test_split(X_normalized, y, test_size=0.3, random_state=np.random.randint(0, 1000))

    # 2. Build the model
    model = Sequential()
    model.add(Dense(10, activation='relu', input_shape=(X_train.shape[1],)))
    model.add(Dense(10, activation='relu'))
    model.add(Dense(10, activation='relu'))
    model.add(Dense(1))


    # Compile the model
    model.compile(optimizer='adam', loss='mean_squared_error')

    # Train the model
    model.fit(X_train, y_train, validation_data=(X_test, y_test), epochs=50, batch_size=10, verbose=2)

    # 3. Evaluate the model
    y_pred = model.predict(X_test)
    mse = mean_squared_error(y_test, y_pred)

    # Store MSE
    mse_list.append(mse)

Epoch 1/50


/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 21ms/step - loss: 1505.8994 - val_loss: 1317.2885
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1262.6868 - val_loss: 943.8519
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 715.9786 - val_loss: 340.0561
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 258.2314 - val_loss: 188.1956
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 189.9649 - val_loss: 174.1737
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 176.8622 - val_loss: 166.1562
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 166.1788 - val_loss: 160.1483
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 158.7333 - val_loss: 156.1684
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 151.0989 - val_loss: 153.1118
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 147.1145 - val_loss: 149.2652
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 141.0209 - val_loss: 147.1583
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 136.6937 - val_loss: 146.3932
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 133.1701 - val_loss: 143.3333
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 130.5363 - val_loss: 140.7836
Epoch 15

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1486.1641 - val_loss: 1516.0913
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1272.5118 - val_loss: 1087.4128
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 661.3558 - val_loss: 367.7862
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 289.2498 - val_loss: 247.3819
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 234.2624 - val_loss: 222.6626
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 205.9969 - val_loss: 207.7167
Epoch 7/50
73/73 - 0s - 5ms/step - loss: 187.0291 - val_loss: 195.7152
Epoch 8/50
73/73 - 0s - 2ms/step - loss: 173.1800 - val_loss: 183.9857
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 160.7202 - val_loss: 171.3642
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 150.1259 - val_loss: 160.1627
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 140.9297 - val_loss: 152.6114
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 132.0212 - val_loss: 143.2722
Epoch 13/50
73/73 - 0s - 2ms/step - loss: 124.0024 - val_loss: 134.3863
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 116.5856 - val_loss: 129.8906
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 19ms/step - loss: 1538.7793 - val_loss: 1498.9908
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1437.3405 - val_loss: 1334.6785
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1127.9547 - val_loss: 876.2524
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 579.9578 - val_loss: 385.8315
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 299.0870 - val_loss: 237.1920
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 213.0663 - val_loss: 188.6163
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 182.7321 - val_loss: 175.6210
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 171.4757 - val_loss: 170.1462
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 167.4039 - val_loss: 166.8808
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 161.6340 - val_loss: 164.1866
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 158.9575 - val_loss: 161.3110
Epoch 12/50
73/73 - 0s - 5ms/step - loss: 155.5915 - val_loss: 159.3671
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 153.0027 - val_loss: 158.7578
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 151.1933 - val_loss: 156.6036
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 19ms/step - loss: 1570.1096 - val_loss: 1392.1774
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1438.6133 - val_loss: 1166.3710
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1037.3025 - val_loss: 613.3441
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 499.1335 - val_loss: 343.7555
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 302.4825 - val_loss: 249.4734
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 230.0397 - val_loss: 203.0081
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 190.6690 - val_loss: 174.4972
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 169.8383 - val_loss: 159.5086
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 157.2840 - val_loss: 150.0781
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 149.2382 - val_loss: 144.5609
Epoch 11/50
73/73 - 1s - 7ms/step - loss: 143.9654 - val_loss: 140.2631
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 139.8213 - val_loss: 134.9435
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 137.1311 - val_loss: 132.5956
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 132.7385 - val_loss: 130.3059
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 28ms/step - loss: 1580.1481 - val_loss: 1550.7839
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1479.1915 - val_loss: 1408.5613
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1218.5380 - val_loss: 1008.4169
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 711.4240 - val_loss: 518.1541
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 382.8268 - val_loss: 324.6021
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 265.6858 - val_loss: 241.7051
Epoch 7/50
73/73 - 0s - 5ms/step - loss: 216.0242 - val_loss: 206.3946
Epoch 8/50
73/73 - 0s - 7ms/step - loss: 196.8026 - val_loss: 190.5942
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 186.9878 - val_loss: 180.3569
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 178.0252 - val_loss: 171.2181
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 169.8054 - val_loss: 162.9980
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 163.4809 - val_loss: 155.4522
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 158.1681 - val_loss: 150.1824
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 152.8967 - val_loss: 146.0801
Epoch

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 29ms/step - loss: 1440.3533 - val_loss: 1400.1420
Epoch 2/50
73/73 - 0s - 5ms/step - loss: 1191.8439 - val_loss: 1013.6006
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 695.8380 - val_loss: 464.8740
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 323.0641 - val_loss: 264.1530
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 225.9942 - val_loss: 241.4799
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 195.6546 - val_loss: 231.2511
Epoch 7/50
73/73 - 0s - 2ms/step - loss: 182.6346 - val_loss: 224.7051
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 174.0644 - val_loss: 216.1278
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 164.9163 - val_loss: 205.5513
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 157.7924 - val_loss: 198.7358
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 151.2569 - val_loss: 191.3300
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 145.2298 - val_loss: 184.1805
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 139.8535 - val_loss: 175.5217
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 133.8928 - val_loss: 168.3552
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


Epoch 1/50
73/73 - 2s - 23ms/step - loss: 1497.3778 - val_loss: 1622.5413
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1414.8182 - val_loss: 1460.5225
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1131.6583 - val_loss: 973.2579
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 595.3037 - val_loss: 415.1605
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 286.4306 - val_loss: 227.0786
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 213.6941 - val_loss: 194.2354
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 195.3582 - val_loss: 177.4874
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 186.2682 - val_loss: 169.0350
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 178.0845 - val_loss: 162.2156
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 171.6579 - val_loss: 157.1094
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 166.2619 - val_loss: 149.4307
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 160.9441 - val_loss: 147.2456
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 157.3159 - val_loss: 141.5592
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 153.8386 - val_loss: 138.

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 23ms/step - loss: 1555.4232 - val_loss: 1491.7350
Epoch 2/50
73/73 - 1s - 7ms/step - loss: 1431.4313 - val_loss: 1287.3904
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1069.5443 - val_loss: 696.8345
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 420.3119 - val_loss: 260.3828
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 242.3736 - val_loss: 212.2918
Epoch 6/50
73/73 - 1s - 9ms/step - loss: 208.5435 - val_loss: 195.0413
Epoch 7/50
73/73 - 1s - 8ms/step - loss: 192.6060 - val_loss: 183.4917
Epoch 8/50
73/73 - 1s - 8ms/step - loss: 183.2999 - val_loss: 175.1768
Epoch 9/50
73/73 - 1s - 7ms/step - loss: 175.3766 - val_loss: 169.4674
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 169.3923 - val_loss: 164.9016
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 164.5673 - val_loss: 159.0201
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 161.1771 - val_loss: 155.5590
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 157.6300 - val_loss: 152.7761
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 154.1877 - val_loss: 150.4857
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 21ms/step - loss: 1522.5520 - val_loss: 1579.7089
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1418.5242 - val_loss: 1410.5181
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1136.0698 - val_loss: 979.7311
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 615.7493 - val_loss: 418.4885
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 287.9592 - val_loss: 261.7080
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 230.1245 - val_loss: 233.1749
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 208.0976 - val_loss: 216.4644
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 195.4033 - val_loss: 206.6537
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 185.5195 - val_loss: 197.9960
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 177.0269 - val_loss: 190.8555
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 170.1098 - val_loss: 184.6155
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 164.6362 - val_loss: 180.7029
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 159.1382 - val_loss: 173.8687
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 154.8230 - val_loss: 169.6669
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1560.1304 - val_loss: 1528.1162
Epoch 2/50
73/73 - 1s - 7ms/step - loss: 1463.1807 - val_loss: 1332.2997
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1025.7188 - val_loss: 632.0876
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 385.3854 - val_loss: 277.7732
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 247.4438 - val_loss: 226.6044
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 215.3678 - val_loss: 205.2872
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 197.3019 - val_loss: 192.5907
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 183.6954 - val_loss: 182.3484
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 173.2047 - val_loss: 173.5399
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 163.6060 - val_loss: 166.9194
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 155.8499 - val_loss: 160.1053
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 149.0536 - val_loss: 154.2841
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 142.8615 - val_loss: 150.2747
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 137.4040 - val_loss: 145.5470
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 21ms/step - loss: 1542.3973 - val_loss: 1447.0734
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1414.4349 - val_loss: 1233.7988
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1057.5756 - val_loss: 744.0494
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 566.8931 - val_loss: 369.4634
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 335.0097 - val_loss: 260.8812
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 244.6372 - val_loss: 221.0277
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 208.4382 - val_loss: 203.7474
Epoch 8/50
73/73 - 0s - 2ms/step - loss: 189.9121 - val_loss: 192.5704
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 177.3188 - val_loss: 182.3206
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 168.9427 - val_loss: 174.0266
Epoch 11/50
73/73 - 1s - 9ms/step - loss: 161.2556 - val_loss: 168.0031
Epoch 12/50
73/73 - 1s - 8ms/step - loss: 155.6682 - val_loss: 162.2952
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 151.8686 - val_loss: 156.9551
Epoch 14/50
73/73 - 1s - 8ms/step - loss: 146.6520 - val_loss: 151.7188
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 31ms/step - loss: 1590.3582 - val_loss: 1491.7030
Epoch 2/50
73/73 - 0s - 4ms/step - loss: 1539.5771 - val_loss: 1428.5726
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1398.9867 - val_loss: 1166.5963
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 933.8053 - val_loss: 571.0452
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 405.9818 - val_loss: 286.4438
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 273.0486 - val_loss: 226.9232
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 236.4084 - val_loss: 200.3719
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 216.3869 - val_loss: 183.1349
Epoch 9/50
73/73 - 1s - 7ms/step - loss: 203.3232 - val_loss: 173.2264
Epoch 10/50
73/73 - 0s - 2ms/step - loss: 192.4150 - val_loss: 165.2708
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 185.2605 - val_loss: 159.2685
Epoch 12/50
73/73 - 0s - 2ms/step - loss: 178.9672 - val_loss: 154.5014
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 172.9082 - val_loss: 150.3080
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 168.4717 - val_loss: 146.6952
Epoch

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 29ms/step - loss: 1562.5841 - val_loss: 1409.0396
Epoch 2/50
73/73 - 0s - 5ms/step - loss: 1382.6930 - val_loss: 1085.0039
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 834.1683 - val_loss: 421.9523
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 338.1584 - val_loss: 235.2704
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 236.7883 - val_loss: 198.3772
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 202.4039 - val_loss: 181.5533
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 183.7597 - val_loss: 169.7266
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 171.2867 - val_loss: 158.5618
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 161.3115 - val_loss: 149.4119
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 152.6842 - val_loss: 142.8055
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 144.6974 - val_loss: 135.8357
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 138.7981 - val_loss: 128.2142
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 132.0801 - val_loss: 123.0038
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 127.4392 - val_loss: 118.1948
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 22ms/step - loss: 1551.4974 - val_loss: 1535.7759
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1460.5635 - val_loss: 1395.9033
Epoch 3/50
73/73 - 0s - 2ms/step - loss: 1213.2360 - val_loss: 1000.9470
Epoch 4/50
73/73 - 0s - 2ms/step - loss: 724.6281 - val_loss: 481.0985
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 357.0761 - val_loss: 283.8044
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 236.2923 - val_loss: 240.7633
Epoch 7/50
73/73 - 0s - 5ms/step - loss: 196.2804 - val_loss: 218.4558
Epoch 8/50
73/73 - 0s - 2ms/step - loss: 173.2780 - val_loss: 202.0597
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 158.7835 - val_loss: 188.4589
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 148.5859 - val_loss: 179.0521
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 139.6417 - val_loss: 168.8258
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 131.3405 - val_loss: 158.3746
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 124.2119 - val_loss: 150.6095
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 118.3349 - val_loss: 143.7678
Epoch

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1585.5371 - val_loss: 1523.2550
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1547.7528 - val_loss: 1480.2634
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1475.5219 - val_loss: 1357.2045
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 1241.2229 - val_loss: 978.5496
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 720.3710 - val_loss: 427.0587
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 301.1531 - val_loss: 266.0192
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 222.9118 - val_loss: 231.1211
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 195.0976 - val_loss: 213.1304
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 179.1692 - val_loss: 201.5979
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 168.3835 - val_loss: 192.9044
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 159.8581 - val_loss: 185.3983
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 153.0276 - val_loss: 178.8260
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 146.3713 - val_loss: 172.4930
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 141.5229 - val_loss: 167.1107
Epoc

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 19ms/step - loss: 1550.9502 - val_loss: 1456.7137
Epoch 2/50
73/73 - 1s - 7ms/step - loss: 1430.3090 - val_loss: 1231.8588
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 987.8702 - val_loss: 578.5991
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 370.8212 - val_loss: 257.3398
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 200.6447 - val_loss: 209.7864
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 173.5275 - val_loss: 195.8377
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 165.7372 - val_loss: 188.3880
Epoch 8/50
73/73 - 0s - 2ms/step - loss: 158.2558 - val_loss: 181.4455
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 154.9501 - val_loss: 178.5756
Epoch 10/50
73/73 - 0s - 2ms/step - loss: 151.4019 - val_loss: 173.9010
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 148.5726 - val_loss: 169.3335
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 146.6064 - val_loss: 166.7428
Epoch 13/50
73/73 - 1s - 8ms/step - loss: 144.2237 - val_loss: 164.3561
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 142.1067 - val_loss: 162.3486
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 23ms/step - loss: 1437.5310 - val_loss: 1269.4850
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1077.6383 - val_loss: 752.0024
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 547.2596 - val_loss: 347.3813
Epoch 4/50
73/73 - 1s - 8ms/step - loss: 296.7697 - val_loss: 254.0845
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 222.2734 - val_loss: 217.5376
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 191.0483 - val_loss: 199.1996
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 174.5075 - val_loss: 184.9845
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 164.7903 - val_loss: 180.7526
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 158.7617 - val_loss: 173.1771
Epoch 10/50
73/73 - 0s - 6ms/step - loss: 152.4497 - val_loss: 167.5900
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 148.6521 - val_loss: 164.9494
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 144.8448 - val_loss: 163.3665
Epoch 13/50
73/73 - 0s - 2ms/step - loss: 141.9540 - val_loss: 159.3743
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 139.7613 - val_loss: 156.0813
Epoch 15

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 26ms/step - loss: 1535.5227 - val_loss: 1402.5177
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1359.9489 - val_loss: 1099.1953
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 841.6887 - val_loss: 441.3757
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 295.0604 - val_loss: 195.1717
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 210.5683 - val_loss: 178.4104
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 192.2032 - val_loss: 166.6197
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 180.3874 - val_loss: 158.4326
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 169.9230 - val_loss: 152.3999
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 162.5361 - val_loss: 147.7359
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 157.3137 - val_loss: 143.1474
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 152.2161 - val_loss: 139.4776
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 148.8645 - val_loss: 135.8855
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 144.9495 - val_loss: 133.7981
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 141.9948 - val_loss: 131.1257
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 21ms/step - loss: 1528.2340 - val_loss: 1553.4260
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1413.9678 - val_loss: 1323.9944
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1012.7902 - val_loss: 744.4945
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 529.2842 - val_loss: 402.1525
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 331.4588 - val_loss: 285.2207
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 254.8068 - val_loss: 236.3175
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 220.1114 - val_loss: 212.6856
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 202.1026 - val_loss: 198.8029
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 189.1975 - val_loss: 188.4358
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 178.8936 - val_loss: 179.2830
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 169.6275 - val_loss: 172.7083
Epoch 12/50
73/73 - 0s - 5ms/step - loss: 162.8713 - val_loss: 165.9518
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 157.5437 - val_loss: 160.7466
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 152.6954 - val_loss: 156.7221
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1481.0322 - val_loss: 1382.9900
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1215.7146 - val_loss: 941.0546
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 638.6550 - val_loss: 357.2754
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 278.6699 - val_loss: 238.3952
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 210.7866 - val_loss: 208.3074
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 186.4036 - val_loss: 192.4364
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 174.6752 - val_loss: 183.5477
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 167.0425 - val_loss: 176.5392
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 161.4491 - val_loss: 171.2098
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 156.3135 - val_loss: 166.3571
Epoch 11/50
73/73 - 0s - 2ms/step - loss: 151.4662 - val_loss: 162.2351
Epoch 12/50
73/73 - 0s - 6ms/step - loss: 148.1540 - val_loss: 157.9801
Epoch 13/50
73/73 - 1s - 8ms/step - loss: 144.9818 - val_loss: 154.5616
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 142.1916 - val_loss: 152.5426
Epoch 15

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1482.5723 - val_loss: 1435.0748
Epoch 2/50
73/73 - 1s - 8ms/step - loss: 1219.8157 - val_loss: 975.0577
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 611.9119 - val_loss: 357.3895
Epoch 4/50
73/73 - 1s - 8ms/step - loss: 277.9288 - val_loss: 272.3953
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 233.4584 - val_loss: 248.0303
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 210.9066 - val_loss: 227.7074
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 196.2102 - val_loss: 213.6584
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 185.0118 - val_loss: 201.6163
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 175.7738 - val_loss: 192.0739
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 167.5259 - val_loss: 183.4188
Epoch 11/50
73/73 - 1s - 8ms/step - loss: 159.6586 - val_loss: 174.3495
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 153.3892 - val_loss: 167.5073
Epoch 13/50
73/73 - 1s - 7ms/step - loss: 146.6566 - val_loss: 158.2366
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 141.5025 - val_loss: 153.3564
Epoch 15

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 29ms/step - loss: 1511.2980 - val_loss: 1435.6395
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1273.1649 - val_loss: 1047.9202
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 698.8973 - val_loss: 413.5486
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 267.3148 - val_loss: 239.8412
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 206.2299 - val_loss: 209.2468
Epoch 6/50
73/73 - 0s - 2ms/step - loss: 186.2265 - val_loss: 194.3507
Epoch 7/50
73/73 - 0s - 5ms/step - loss: 174.1758 - val_loss: 186.7031
Epoch 8/50
73/73 - 0s - 2ms/step - loss: 166.1677 - val_loss: 178.4759
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 159.7067 - val_loss: 172.4808
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 155.4539 - val_loss: 168.0276
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 151.2028 - val_loss: 168.5259
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 147.8902 - val_loss: 162.9553
Epoch 13/50
73/73 - 0s - 2ms/step - loss: 144.6269 - val_loss: 163.0034
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 142.2252 - val_loss: 158.2072
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1563.2451 - val_loss: 1440.1570
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1398.3341 - val_loss: 1222.9744
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1069.4077 - val_loss: 751.1047
Epoch 4/50
73/73 - 0s - 2ms/step - loss: 514.1075 - val_loss: 262.9473
Epoch 5/50
73/73 - 0s - 2ms/step - loss: 228.4138 - val_loss: 198.5353
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 193.3884 - val_loss: 182.4122
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 179.6504 - val_loss: 173.1172
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 170.1533 - val_loss: 166.3300
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 161.4136 - val_loss: 160.4890
Epoch 10/50
73/73 - 0s - 2ms/step - loss: 155.3275 - val_loss: 156.5396
Epoch 11/50
73/73 - 0s - 2ms/step - loss: 148.9811 - val_loss: 152.9653
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 143.3631 - val_loss: 150.1451
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 139.1342 - val_loss: 146.4937
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 135.7243 - val_loss: 144.3925
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1511.5776 - val_loss: 1466.6302
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1321.4270 - val_loss: 1109.2532
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 789.3762 - val_loss: 499.1728
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 354.4799 - val_loss: 296.2213
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 246.0219 - val_loss: 236.9904
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 209.0274 - val_loss: 207.9122
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 187.1784 - val_loss: 190.6111
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 174.5258 - val_loss: 178.0737
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 165.5202 - val_loss: 168.2992
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 159.5893 - val_loss: 162.2724
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 154.1780 - val_loss: 156.5193
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 152.4057 - val_loss: 154.0442
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 146.0182 - val_loss: 150.0096
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 143.1439 - val_loss: 145.7442
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 21ms/step - loss: 1543.1489 - val_loss: 1427.9099
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1373.1309 - val_loss: 1118.8823
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 855.7089 - val_loss: 427.4196
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 306.6716 - val_loss: 195.9253
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 204.4315 - val_loss: 165.0457
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 177.4298 - val_loss: 153.5695
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 168.5233 - val_loss: 149.4473
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 160.8686 - val_loss: 145.9025
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 156.0201 - val_loss: 143.5658
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 152.6769 - val_loss: 141.8027
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 149.6290 - val_loss: 140.0666
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 147.7509 - val_loss: 138.5742
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 146.0264 - val_loss: 135.8561
Epoch 14/50
73/73 - 0s - 2ms/step - loss: 144.7238 - val_loss: 135.1258
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1523.0409 - val_loss: 1509.0906
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1381.3809 - val_loss: 1294.8438
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1051.1757 - val_loss: 847.2025
Epoch 4/50
73/73 - 0s - 2ms/step - loss: 554.2075 - val_loss: 388.8727
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 272.3622 - val_loss: 253.2970
Epoch 6/50
73/73 - 0s - 2ms/step - loss: 208.4195 - val_loss: 215.7809
Epoch 7/50
73/73 - 0s - 2ms/step - loss: 181.7292 - val_loss: 195.1725
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 167.0397 - val_loss: 182.5799
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 157.5793 - val_loss: 173.9443
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 149.9828 - val_loss: 168.1959
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 143.2311 - val_loss: 160.6879
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 137.3785 - val_loss: 156.0407
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 133.2501 - val_loss: 149.2242
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 128.6397 - val_loss: 145.4838
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1543.2262 - val_loss: 1377.9244
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1328.9373 - val_loss: 1039.6218
Epoch 3/50
73/73 - 0s - 2ms/step - loss: 828.9494 - val_loss: 483.2816
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 388.6772 - val_loss: 280.0454
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 270.5976 - val_loss: 226.9507
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 228.4926 - val_loss: 200.1666
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 204.5651 - val_loss: 183.7065
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 186.5283 - val_loss: 171.4527
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 172.3997 - val_loss: 159.5056
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 159.4019 - val_loss: 150.0878
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 148.7096 - val_loss: 142.7587
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 139.3458 - val_loss: 136.0975
Epoch 13/50
73/73 - 1s - 9ms/step - loss: 131.1163 - val_loss: 129.1027
Epoch 14/50
73/73 - 1s - 8ms/step - loss: 123.9989 - val_loss: 123.1048
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 29ms/step - loss: 1539.1117 - val_loss: 1483.9159
Epoch 2/50
73/73 - 0s - 5ms/step - loss: 1333.7742 - val_loss: 1160.9467
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 838.3290 - val_loss: 541.0085
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 346.9633 - val_loss: 281.2251
Epoch 5/50
73/73 - 1s - 9ms/step - loss: 239.8293 - val_loss: 239.7989
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 210.2247 - val_loss: 219.9152
Epoch 7/50
73/73 - 1s - 7ms/step - loss: 193.0027 - val_loss: 206.9575
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 182.5143 - val_loss: 196.3002
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 173.4897 - val_loss: 188.0525
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 167.0406 - val_loss: 182.0559
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 161.1012 - val_loss: 175.4852
Epoch 12/50
73/73 - 0s - 5ms/step - loss: 155.6170 - val_loss: 172.5404
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 150.9379 - val_loss: 166.6722
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 146.5173 - val_loss: 163.1309
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 26ms/step - loss: 1541.5219 - val_loss: 1584.0840
Epoch 2/50
73/73 - 1s - 7ms/step - loss: 1454.6500 - val_loss: 1418.7561
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1120.9865 - val_loss: 860.9235
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 470.5246 - val_loss: 265.9759
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 213.1820 - val_loss: 197.3002
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 189.4786 - val_loss: 178.4289
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 177.4339 - val_loss: 169.1255
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 169.0430 - val_loss: 158.4181
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 162.8256 - val_loss: 156.4268
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 158.0978 - val_loss: 147.6554
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 154.1108 - val_loss: 144.4373
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 150.0681 - val_loss: 140.2829
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 146.3048 - val_loss: 137.3747
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 143.4161 - val_loss: 132.9618
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 22ms/step - loss: 1559.1478 - val_loss: 1481.2164
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1438.8766 - val_loss: 1266.0924
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1060.2352 - val_loss: 741.3480
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 483.0146 - val_loss: 328.9800
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 254.4428 - val_loss: 244.7881
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 202.4313 - val_loss: 210.0334
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 183.3559 - val_loss: 193.9531
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 175.1862 - val_loss: 185.7135
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 164.5499 - val_loss: 177.0752
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 158.4476 - val_loss: 169.9882
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 153.4123 - val_loss: 164.6296
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 147.5787 - val_loss: 159.0776
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 142.0988 - val_loss: 153.1485
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 136.7893 - val_loss: 147.6524
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 22ms/step - loss: 1579.5385 - val_loss: 1527.6313
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1555.9828 - val_loss: 1504.7830
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 1519.1135 - val_loss: 1440.7952
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1395.6750 - val_loss: 1227.3987
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 1029.2233 - val_loss: 684.5114
Epoch 6/50
73/73 - 1s - 9ms/step - loss: 488.0756 - val_loss: 294.7963
Epoch 7/50
73/73 - 1s - 9ms/step - loss: 292.8349 - val_loss: 209.2529
Epoch 8/50
73/73 - 0s - 6ms/step - loss: 232.8295 - val_loss: 175.7642
Epoch 9/50
73/73 - 1s - 8ms/step - loss: 204.1126 - val_loss: 159.1880
Epoch 10/50
73/73 - 1s - 7ms/step - loss: 185.3865 - val_loss: 148.8165
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 173.3946 - val_loss: 139.9163
Epoch 12/50
73/73 - 1s - 7ms/step - loss: 163.6070 - val_loss: 135.0065
Epoch 13/50
73/73 - 0s - 2ms/step - loss: 156.5674 - val_loss: 131.2107
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 151.4128 - val_loss: 128.7045
Ep

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 32ms/step - loss: 1499.5309 - val_loss: 1461.6064
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1313.9178 - val_loss: 1162.1960
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 887.5696 - val_loss: 665.5067
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 479.8716 - val_loss: 343.5105
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 264.7509 - val_loss: 218.2607
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 192.1304 - val_loss: 187.3492
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 175.4603 - val_loss: 176.9708
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 164.4950 - val_loss: 167.7679
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 157.2675 - val_loss: 161.4138
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 152.1279 - val_loss: 155.7083
Epoch 11/50
73/73 - 0s - 2ms/step - loss: 147.5118 - val_loss: 151.3313
Epoch 12/50
73/73 - 0s - 2ms/step - loss: 143.2195 - val_loss: 147.2818
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 140.3404 - val_loss: 143.7364
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 136.5216 - val_loss: 140.5298
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 22ms/step - loss: 1459.3456 - val_loss: 1584.8265
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1286.6451 - val_loss: 1294.4661
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 881.9785 - val_loss: 738.3698
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 411.5869 - val_loss: 366.2423
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 264.9791 - val_loss: 275.1789
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 219.8305 - val_loss: 237.1302
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 196.7059 - val_loss: 214.3436
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 183.9914 - val_loss: 200.1720
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 173.4703 - val_loss: 190.6123
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 166.7747 - val_loss: 184.6264
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 160.4512 - val_loss: 174.8280
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 155.8526 - val_loss: 170.1489
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 150.9006 - val_loss: 165.5233
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 147.3564 - val_loss: 162.3448
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 22ms/step - loss: 1529.8519 - val_loss: 1382.8329
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1353.7251 - val_loss: 1099.5842
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 928.7668 - val_loss: 583.4659
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 431.1954 - val_loss: 253.9585
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 251.5010 - val_loss: 209.1955
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 214.4330 - val_loss: 192.0545
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 194.9076 - val_loss: 182.8798
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 183.3228 - val_loss: 174.9826
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 173.8894 - val_loss: 168.8377
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 166.7250 - val_loss: 164.3760
Epoch 11/50
73/73 - 0s - 6ms/step - loss: 160.7933 - val_loss: 159.7105
Epoch 12/50
73/73 - 1s - 8ms/step - loss: 155.3448 - val_loss: 158.1553
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 150.1877 - val_loss: 154.4715
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 146.1636 - val_loss: 151.4932
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 33ms/step - loss: 1500.3610 - val_loss: 1509.8021
Epoch 2/50
73/73 - 0s - 5ms/step - loss: 1380.3473 - val_loss: 1306.5496
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 1060.6123 - val_loss: 842.3822
Epoch 4/50
73/73 - 1s - 8ms/step - loss: 551.1512 - val_loss: 381.7440
Epoch 5/50
73/73 - 1s - 8ms/step - loss: 286.3540 - val_loss: 272.5473
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 238.6857 - val_loss: 236.1942
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 213.9928 - val_loss: 213.6651
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 198.4614 - val_loss: 196.1002
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 186.0299 - val_loss: 182.8857
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 177.0326 - val_loss: 173.1348
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 170.4888 - val_loss: 164.9708
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 164.9269 - val_loss: 158.3389
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 159.3722 - val_loss: 152.9699
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 154.9310 - val_loss: 147.5277
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 22ms/step - loss: 1449.6847 - val_loss: 1391.3077
Epoch 2/50
73/73 - 1s - 7ms/step - loss: 1175.5638 - val_loss: 924.4846
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 593.9848 - val_loss: 361.7414
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 257.2755 - val_loss: 246.7512
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 202.3184 - val_loss: 214.7592
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 182.1502 - val_loss: 203.0650
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 171.6279 - val_loss: 195.8231
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 163.4221 - val_loss: 185.0672
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 155.7428 - val_loss: 178.8922
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 149.6956 - val_loss: 175.1415
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 146.1864 - val_loss: 170.9809
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 140.6522 - val_loss: 166.0588
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 137.1106 - val_loss: 162.2396
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 133.0708 - val_loss: 163.8562
Epoch 15

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 6s - 78ms/step - loss: 1485.0619 - val_loss: 1521.2965
Epoch 2/50
73/73 - 1s - 8ms/step - loss: 1290.3821 - val_loss: 1162.0038
Epoch 3/50
73/73 - 1s - 9ms/step - loss: 741.3705 - val_loss: 436.8970
Epoch 4/50
73/73 - 0s - 6ms/step - loss: 277.5627 - val_loss: 242.5595
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 194.3945 - val_loss: 210.7592
Epoch 6/50
73/73 - 1s - 7ms/step - loss: 172.2318 - val_loss: 197.5699
Epoch 7/50
73/73 - 0s - 6ms/step - loss: 163.7466 - val_loss: 189.8793
Epoch 8/50
73/73 - 1s - 7ms/step - loss: 156.4625 - val_loss: 184.6774
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 152.1294 - val_loss: 179.8888
Epoch 10/50
73/73 - 1s - 7ms/step - loss: 148.5237 - val_loss: 176.1384
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 144.4047 - val_loss: 173.0246
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 141.1271 - val_loss: 169.3080
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 139.0035 - val_loss: 166.1307
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 135.2923 - val_loss: 163.4922
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 31ms/step - loss: 1490.5215 - val_loss: 1332.3521
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1252.1045 - val_loss: 975.0255
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 771.9619 - val_loss: 435.4083
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 367.6416 - val_loss: 264.5958
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 268.3406 - val_loss: 227.1282
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 238.7991 - val_loss: 202.4500
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 216.2023 - val_loss: 181.9612
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 195.8744 - val_loss: 165.5974
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 180.0183 - val_loss: 149.9572
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 165.9334 - val_loss: 138.5145
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 153.5036 - val_loss: 127.8894
Epoch 12/50
73/73 - 0s - 5ms/step - loss: 143.0962 - val_loss: 118.7544
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 133.4855 - val_loss: 110.8504
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 125.7817 - val_loss: 103.8966
Epoch 15

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 23ms/step - loss: 1546.1490 - val_loss: 1415.7744
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1371.1268 - val_loss: 1130.9608
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 942.1394 - val_loss: 620.0450
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 479.4518 - val_loss: 285.4322
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 256.0122 - val_loss: 184.7517
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 201.8784 - val_loss: 165.8921
Epoch 7/50
73/73 - 0s - 5ms/step - loss: 184.5009 - val_loss: 156.4766
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 173.7065 - val_loss: 150.7163
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 166.0693 - val_loss: 146.4278
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 159.9944 - val_loss: 142.5255
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 155.0906 - val_loss: 140.0652
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 152.2046 - val_loss: 137.5363
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 146.8785 - val_loss: 135.8073
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 143.9927 - val_loss: 133.8062
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 23ms/step - loss: 1536.1047 - val_loss: 1379.4482
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1318.2980 - val_loss: 977.7843
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 693.4823 - val_loss: 311.2722
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 232.2621 - val_loss: 227.4119
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 197.7231 - val_loss: 208.5542
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 185.3114 - val_loss: 201.7971
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 175.4926 - val_loss: 192.6391
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 168.8460 - val_loss: 187.6470
Epoch 9/50
73/73 - 0s - 6ms/step - loss: 161.8256 - val_loss: 181.0350
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 156.1010 - val_loss: 173.7118
Epoch 11/50
73/73 - 1s - 9ms/step - loss: 152.2576 - val_loss: 167.5187
Epoch 12/50
73/73 - 1s - 8ms/step - loss: 148.8336 - val_loss: 168.6864
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 144.3357 - val_loss: 163.9204
Epoch 14/50
73/73 - 1s - 9ms/step - loss: 141.9422 - val_loss: 163.0630
Epoch 15

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 29ms/step - loss: 1525.2598 - val_loss: 1387.5250
Epoch 2/50
73/73 - 0s - 5ms/step - loss: 1322.8243 - val_loss: 1049.7307
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 832.7230 - val_loss: 481.2017
Epoch 4/50
73/73 - 0s - 5ms/step - loss: 380.7458 - val_loss: 303.6160
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 250.5210 - val_loss: 245.2631
Epoch 6/50
73/73 - 1s - 9ms/step - loss: 202.9998 - val_loss: 215.5485
Epoch 7/50
73/73 - 1s - 8ms/step - loss: 183.2330 - val_loss: 196.8259
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 169.5655 - val_loss: 187.7741
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 160.0742 - val_loss: 177.7624
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 153.4075 - val_loss: 171.6351
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 146.9821 - val_loss: 165.8274
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 141.1693 - val_loss: 162.1746
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 136.5240 - val_loss: 157.3758
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 131.5978 - val_loss: 153.3002
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 22ms/step - loss: 1564.6469 - val_loss: 1462.9747
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1468.0865 - val_loss: 1299.5951
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1195.6165 - val_loss: 872.5790
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 659.2426 - val_loss: 350.0035
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 291.0597 - val_loss: 256.9865
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 223.9570 - val_loss: 229.2495
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 198.1865 - val_loss: 207.1023
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 182.4526 - val_loss: 200.0381
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 172.3971 - val_loss: 189.1336
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 164.0342 - val_loss: 186.1107
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 157.3229 - val_loss: 177.9816
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 151.6420 - val_loss: 173.0055
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 146.8264 - val_loss: 170.5023
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 143.3868 - val_loss: 167.1318
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 22ms/step - loss: 1523.0065 - val_loss: 1465.7937
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1389.9436 - val_loss: 1253.0223
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1052.6647 - val_loss: 757.0448
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 536.4376 - val_loss: 325.9003
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 279.5762 - val_loss: 238.8571
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 224.1980 - val_loss: 213.6518
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 204.8477 - val_loss: 198.9955
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 191.3493 - val_loss: 189.8035
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 182.8237 - val_loss: 181.3496
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 175.5678 - val_loss: 174.1473
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 169.0193 - val_loss: 167.9964
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 164.0201 - val_loss: 165.2509
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 158.9610 - val_loss: 159.4771
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 155.6334 - val_loss: 156.4638
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 22ms/step - loss: 1572.2513 - val_loss: 1406.9298
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1420.3397 - val_loss: 1144.3240
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 977.4293 - val_loss: 579.9409
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 436.2055 - val_loss: 274.0437
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 257.7360 - val_loss: 217.6934
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 215.1760 - val_loss: 194.0088
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 196.7804 - val_loss: 180.5306
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 184.0337 - val_loss: 171.6430
Epoch 9/50
73/73 - 1s - 7ms/step - loss: 174.5206 - val_loss: 164.1008
Epoch 10/50
73/73 - 1s - 7ms/step - loss: 167.5655 - val_loss: 158.1873
Epoch 11/50
73/73 - 1s - 8ms/step - loss: 161.6073 - val_loss: 152.9537
Epoch 12/50
73/73 - 1s - 8ms/step - loss: 156.6194 - val_loss: 148.2214
Epoch 13/50
73/73 - 1s - 9ms/step - loss: 151.9404 - val_loss: 145.0309
Epoch 14/50
73/73 - 1s - 8ms/step - loss: 147.1768 - val_loss: 142.0328
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 30ms/step - loss: 1479.2438 - val_loss: 1434.2958
Epoch 2/50
73/73 - 0s - 4ms/step - loss: 1271.7889 - val_loss: 1110.5898
Epoch 3/50
73/73 - 1s - 9ms/step - loss: 813.7877 - val_loss: 550.4459
Epoch 4/50
73/73 - 0s - 6ms/step - loss: 384.0159 - val_loss: 279.7552
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 242.9586 - val_loss: 202.3947
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 196.2454 - val_loss: 175.2424
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 180.6810 - val_loss: 166.9754
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 171.5791 - val_loss: 161.3698
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 166.5581 - val_loss: 156.5952
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 161.1406 - val_loss: 153.5259
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 157.9391 - val_loss: 150.2163
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 154.6960 - val_loss: 148.5358
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 151.5123 - val_loss: 146.2929
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 148.9703 - val_loss: 145.3492
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 22ms/step - loss: 1576.3523 - val_loss: 1437.3317
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1433.9612 - val_loss: 1186.8287
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 997.7737 - val_loss: 628.7172
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 529.5381 - val_loss: 393.2623
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 370.0510 - val_loss: 308.4681
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 293.5557 - val_loss: 254.5607
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 242.1096 - val_loss: 224.7828
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 210.7329 - val_loss: 203.0458
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 187.8381 - val_loss: 190.1564
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 175.6357 - val_loss: 180.2668
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 164.0617 - val_loss: 175.2432
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 155.6421 - val_loss: 168.1621
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 150.0371 - val_loss: 163.5210
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 145.7418 - val_loss: 159.2053
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 21ms/step - loss: 1458.4003 - val_loss: 1374.4440
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1209.6024 - val_loss: 1009.5975
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 759.8672 - val_loss: 539.4509
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 388.9202 - val_loss: 301.7611
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 249.1667 - val_loss: 229.8027
Epoch 6/50
73/73 - 0s - 2ms/step - loss: 211.8191 - val_loss: 203.8874
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 194.5065 - val_loss: 187.8632
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 182.7274 - val_loss: 173.9329
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 171.9864 - val_loss: 162.2565
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 163.7555 - val_loss: 152.7510
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 154.5205 - val_loss: 142.2766
Epoch 12/50
73/73 - 0s - 5ms/step - loss: 146.4530 - val_loss: 134.0079
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 139.0670 - val_loss: 128.6101
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 133.3379 - val_loss: 120.6629
Epoch 1

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 21ms/step - loss: 1566.5182 - val_loss: 1480.5426
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1482.6881 - val_loss: 1318.7236
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1126.2678 - val_loss: 810.2127
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 579.8899 - val_loss: 417.5441
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 364.0325 - val_loss: 312.0431
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 288.9058 - val_loss: 260.3759
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 241.8251 - val_loss: 224.8512
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 212.2512 - val_loss: 202.8362
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 191.6108 - val_loss: 186.2431
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 177.8095 - val_loss: 177.1209
Epoch 11/50
73/73 - 1s - 8ms/step - loss: 166.6093 - val_loss: 166.5161
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 159.1631 - val_loss: 160.7181
Epoch 13/50
73/73 - 1s - 8ms/step - loss: 152.9480 - val_loss: 156.5889
Epoch 14/50
73/73 - 1s - 8ms/step - loss: 149.5557 - val_loss: 151.8386
Epoch 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 26ms/step - loss: 1523.6469 - val_loss: 1339.2969
Epoch 2/50
73/73 - 0s - 5ms/step - loss: 1252.6680 - val_loss: 920.4640
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 627.8771 - val_loss: 355.3089
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 276.2471 - val_loss: 241.9153
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 213.1995 - val_loss: 206.6999
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 189.4580 - val_loss: 190.7899
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 176.1647 - val_loss: 182.0161
Epoch 8/50
73/73 - 1s - 9ms/step - loss: 166.2872 - val_loss: 174.4772
Epoch 9/50
73/73 - 1s - 7ms/step - loss: 158.0147 - val_loss: 168.0410
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 151.7177 - val_loss: 165.3234
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 146.3868 - val_loss: 161.4747
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 141.8572 - val_loss: 158.4888
Epoch 13/50
73/73 - 0s - 2ms/step - loss: 138.2222 - val_loss: 154.5443
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 135.2991 - val_loss: 151.5223
Epoch 15

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 30ms/step - loss: 1527.3235 - val_loss: 1591.2561
Epoch 2/50
73/73 - 0s - 5ms/step - loss: 1445.8657 - val_loss: 1449.5928
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1190.2998 - val_loss: 1010.4534
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 635.4246 - val_loss: 395.5742
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 299.9289 - val_loss: 267.3428
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 245.0376 - val_loss: 238.0356
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 220.2915 - val_loss: 218.6991
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 203.7531 - val_loss: 202.1758
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 191.5756 - val_loss: 193.9635
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 181.6428 - val_loss: 184.5085
Epoch 11/50
73/73 - 0s - 2ms/step - loss: 174.0432 - val_loss: 176.0578
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 166.9966 - val_loss: 169.9095
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 160.3908 - val_loss: 163.2221
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 156.9799 - val_loss: 158.8455
Epoch

In [41]:
# 4. Calculate mean and standard deviation of MSE
mean_mse = np.mean(mse_list)
std_mse = np.std(mse_list)

In [42]:
# Print results after normalization with 50 epochs and 3 hiddens layer
print(f"Mean of MSE over 50 runs: {mean_mse}")
print(f"Standard Deviation of MSE over 50 runs: {std_mse}")

Mean of MSE over 50 runs: 87.84045185522876
Standard Deviation of MSE over 50 runs: 25.615244409073615


In [29]:
# Print results after normalization with 50 epochs and a hidden layer
print(f"Mean of MSE over 50 runs: {mean_mse}")
print(f"Standard Deviation of MSE over 50 runs: {std_mse}")

Mean of MSE over 50 runs: 130.31662939683622
Standard Deviation of MSE over 50 runs: 11.836501926494584


By increasing the number of hidden layers (from 1 to 3), the Mean MSE significantly decreased (130.316 → 87.840), showing that the model's predictions became more accurate on average.

**However**, the Standard Deviation of MSE increased (11.836 → 25.615), indicating that the performance became less consistent across the 50 runs.